tests/conditionalVault.ts

```diff
@@ -1154,11 +1172,11 @@ export async function redeemDepositSlip(
   assert.equal(
     vaultUnderlyingTokenAccountAfter.amount,
     vaultUnderlyingTokenAccountBefore.amount -
-      BigInt(depositSlipBefore.depositedAmount)
+      BigInt(depositSlipBefore.depositedAmount.toString())
```

Comment:
it's a bummer that there's not something cleaner here, but yeah it appears that this is the best way to do it on a `BN`